File: backend/app/src/main/java/com/yagubogu/member/dto/v1/MemberFavoriteResponse.java
package com.yagubogu.member.dto.v1;

import com.yagubogu.team.domain.Team;

public record MemberFavoriteResponse(
        String favorite
) {

    public static MemberFavoriteResponse from(final Team team) {
        return new MemberFavoriteResponse(team.getShortName());
    }

    public static MemberFavoriteResponse empty() {
        return new MemberFavoriteResponse(null);
    }
}

File: android/app/src/main/java/com/yagubogu/data/dto/response/MemberFavoriteResponse.kt
package com.yagubogu.data.dto.response

import kotlinx.serialization.SerialName
import kotlinx.serialization.Serializable

@Serializable
data class MemberFavoriteResponse(
    @SerialName("favorite")
    val favorite: String? = null, // 팀 이름
)
